asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt

```diff
@@ -329,10 +329,30 @@ abstract class AbstractWrapper(val publisher : PulsarPublisher, val wpsConfigura
         LOGGER.info("Start mapping to wps inputs")
         val wpsInputMapper = InputMapper(getWpsIdentifier())
         val wpsInputs = wpsInputMapper.mapInputs(complexInputs, complexInputsAsValues, complexOutputsAsInputs, literalInputs, bboxInputs)
+        var wpsProcess: WPSProcess? = null
```

Comment:
Does this retry require any more coordination with @arnevogt ? If so, we can talk through that during tomorrow's stand-up. 
Looks fine by me, though!

Reply:
Oh, never mind @nbrinckm : you already said that you wanted to discuss this with @arnevogt . Taking back my comment.